src/util/types.ts

```diff
@@ -1050,13 +1050,35 @@ export interface GeometrySector {
 export type D3Scale<T> = D3ScaleContinuousNumeric<T, number>;
 
 export type AxisDomainItem = string | number | Function | 'auto' | 'dataMin' | 'dataMax';
-/** The domain of axis */
+/**
+ * The domain of axis.
```

Comment:
```suggestion
 * The domain of axis, always defined by an array of exactly two values, for the min and the max of the axis. 
```

Reply:
The only information that is missing here, is that it must the exactly two values

Reply:
Thanks for the confirmation, I wasn't sure if that's the case judging from types alone.

Reply:
Hmmm @nikolasrieble the more I look into `getDomainOfDataByKey` the more I'm wondering. What does categorical domain look like? If it's a bunch of strings then all the strings will be part of the domain definition, right? It's only the numerical domain that can simplify the definition to `[min, max]`.